presentations/en/2021-micro-acquisition-elevator-pitch.md

```diff
@@ -0,0 +1,103 @@
+
+<style>
+.reveal section img { background:none; border:none; box-shadow:none; }
+</style>
+
+# Micro-Acquisition Pilot
+
+ESDC IT Strategy Team
+
+Note:
+
+- we are here today to talk to you about our procurement pilot and we will
+- but first we want to talk a little about what we believe
+
+---
+
+Trying to continuously improve while also keeping the lights on is hard
+
+Note:
+
+---
+
+Learning new things is essential to keep up with the fast pace of IT
+
+Note:
+
+(and we know that learning often takes a back seat to operational activities)
+
+---
+
+Sharing code across teams, branches and departments can free up valuable time
+
+Note:
+
+- reduce duplication
+- participate in a community
+- OSS code is more secure
+
+---
+
+IT contract dollars going to the same usual suspects reduces our exposure to new ideas
+
+Note:
+
+---
+
+There is a vast amount of IT talent out there for whom a $10K contract would mean a whole lot
+
+Note:
+
+---
+
+**add image of lightbulb**
+Fix everything with procurement!
+
+Note:
+
+- joking. This is a complex problem with multiple solutions contribution to moving us forward.
+- Outsourcing large IT solutions has a problematic track record
+- We are not pro-outsourcing all things.
+- Phoenix, canada.ca email examples of outsourcing gone wrong for many reasons
+- but we believe there is a strong benefit to outsourcing when keeping the lights on means that issues that remain stuck in the backlog, or when having no time to learn new skills prevents moving towards modern approaches.
+- procurement could be one piece of the puzzle.
+
+---
+
+*add image of the computer from the micro-acquisition website*
+
+- procurement of custom code
+- licenced as open source
+- <$10K
+- no security clearance required
+- work done in the open
+- paid by credit card
+
+Note:
+
+- so we want to introduce you to the micro-acquisition pilot
+- this is a procurement option developed in partnership between IITB and CFOB
+- through this option you can procure custom code and only custom code (think automation scripts, accessibility and bug fixes, small add/on modules, data analysis). The code that is delivered must be licenced as open source and the total value of the contract must be less than $10K including taxes. The work is all done on open repositories
+- suppliers face reduced barriers because they do not require a security clearance and qualification does not require business income in the millions nor proof of work of other contracts
+
+---
+
+DEMO
+
+*add image of the main page of the site*
+
+---
+
+We are reducing risk through:
+
+- code acceptance criteria
+- taking lessons from others who have done this
+- partnership with CFOB
+
+Note:
+
+- before we finish i want to talk briefly about how we are reducing risk
+- opportunity statement must include the tests we will be doing on the code to ensure it meets functional requirements as well as quality and security requirements
```

Comment:
I suspect we can be asked about security testing and the supply chain (that argument that OSS can be deemed less secure because there is no liability to a vendor).

I suggest framing it as such:
- the same security controls and due diligence that developers are expected to observe remain. We are promoting a security culture by clearly setting expectations that teams are ultimately accountable for the code running in their applications.
- improving security testing in pipelines can benefit from this vehicle by promoting sharing of security tests across the branch and departments, temporary and rapidly increasing a team's capacity by outsourcing some testing scripting work to the public.